lua/esqueleto/utils.lua

```diff
@@ -78,8 +80,20 @@ local listignored = function(dir, ignored_patterns)
     )
   )
 end
-
--- Returns a ignore checker
+-- ---List ignored files under a directory, given a list of glob patterns
+-- ---@param dir string
+-- ---@param ignored_patterns string[]
+-- ---@return string[]
+-- local listignore = function(dir, ignored_patterns)
```

Comment:
Iter alternative to listignored

Reply:
I would add this version but I need to implement proper testing before relying on changing it. Will test it in my local version and decide on wheter we use this one for now.